src/components/DataTable/DataTable.js

```diff
@@ -43,86 +42,30 @@ const DataTable = ({
 
   isSortable,
   isSelectable,
-  missingDataCharacter,
   render,
   rows,
   sortRow,
 
   useZebraStyles,
+  ...other
 }) => (
   <div className={namespace}>
-    {render ? (
-      <CarbonDataTable
-        filterRows={filterRows}
-        headers={headers}
-        isSortable={isSortable}
-        render={render}
-        rows={rows}
-        sortRow={sortRow}
-        useZebraStyles={useZebraStyles}
-      />
-    ) : (
-      <CarbonDataTable
-        headers={headers}
-        rows={rows}
-        filterRows={filterRows}
-        sortRow={sortRow}
-        useZebraStyles={useZebraStyles}
-        render={({
-          rows,
-          headers,
-          getHeaderProps,
-          getRowProps,
-          getSelectionProps,
-          getTableProps,
-        }) => (
-          <TableContainer>
-            <Table {...getTableProps()}>
-              <TableHead>
-                <TableRow>
-                  {isSelectable && <TableSelectAll {...getSelectionProps()} />}
-                  {headers.map(header => (
-                    <TableHeader
-                      {...getHeaderProps({ header, isSortable })}
-                      key={header.key}
-                    >
-                      {header.header}
-                    </TableHeader>
-                  ))}
-                </TableRow>
-              </TableHead>
-              <TableBody>
-                {rows.map(row => (
-                  <TableRow
-                    {...getRowProps({ row })}
-                    className={classnames({
-                      [`${namespace}__row--selectable`]: isSelectable,
-                    })}
-                    key={row.id}
-                  >
-                    {isSelectable && (
-                      <TableSelectRow {...getSelectionProps({ row })} />
-                    )}
-                    {row.cells.map(cell => (
-                      <TableCell key={cell.id}>
-                        {cell.value ? cell.value : missingDataCharacter}
-                      </TableCell>
-                    ))}
-                  </TableRow>
-                ))}
-              </TableBody>
-            </Table>
-          </TableContainer>
-        )}
-      />
-    )}
+    <CarbonDataTable
+      filterRows={filterRows}
```

Comment:
Nice! I wonder here if we can treat this as a true pass-through now and remove all of these props bar `other`, pointing to Carbon's Storybook for additional guidance